content/docs/get-started/policy-as-code/enforcing-a-policy-pack.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Enforcing a Policy Pack Across an Organization (Preview)
+linktitle: Enforcing a Policy Pack
+weight: 2
+menu:
+  getstarted:
+    parent: Policy as Code
+    identifier: policy-as-code-enforcing-a-policy-pack
+
+aliases: ["/docs/quickstart/policy-as-code/enforcing-a-policy-pack/"]
+---
+{{% pac-preview %}}
+
+Once you’ve validated the behavior of your policies, an organization administrator can publish them to the Pulumi Console to be enforced across your organization. Any Pulumi client (a developer’s workstation, CI/CD tool, etc) that interacts with a stack via the Pulumi Console will have policy enforcement during the execution of `preview` and `update`. Policy Packs are versioned by the Pulumi Console so that updated policies can be published and applied as ready and also reverted to previous versions as needed.
+
+1. From within the Policy Pack directory, run the following command to publish your pack:
+
+    ```sh
+    PULUMI_DEBUG_COMMANDS=true pulumi policy publish <organization>/<policy-pack-name>
+    ```
+
+    The <policy-pack-name> is the name you’d like to see used to reference the pack in the Pulumi Console.
+
+    The output will tell you what version of the Policy Pack you just published. The Pulumi service provides a monotonic version number for Policy Packs.
+
+    ```sh
+    $ PULUMI_DEBUG_COMMANDS=true pulumi policy publish pulumi/policy-pack-typescript
+    Obtaining policy metadata from policy plugin
+    Compressing policy pack
+    Uploading policy pack to Pulumi service
+    Publishing as policy-pack-typescript
+    Published as version 1
+    ```
+
+1. You can apply this Policy Pack to your organization’s default Policy Group by running:
+
+    ```sh
+    PULUMI_DEBUG_COMMANDS=true pulumi policy apply <organization>/<policy-pack-name> <version>
+    ```
+
+    For example, to apply the Policy Pack created in the previous step:
+
+    ```sh
+    PULUMI_DEBUG_COMMANDS=true pulumi policy apply pulumi/policy-pack-typescript 1
+    ```
+
+    The CLI can only be used to apply the Policy Pack to your default Policy Group. If you would like to add the Policy Pack to a different Policy Group, you can do so via the Pulumi Console.
```

Comment:
It is strange that the stepper abruptly ends here. I was going down the next steps, and now all of a sudden, there aren't any (yet I see more pages on the left-nav for "Best Practices" and "Core Concepts". Should we continue the stepper for these? If the stepper is visible, then you'll be able to easily click the "Previous Step" if needed.

Otherwise, if we're not going to have the stepper here, perhaps we should consider at the very least having a "Next Steps" summary section that links to Best Practices and Core Concepts.

Reply:
I do have links to the `Core Concepts` and `Best Practices` throughout, but I think you're definitely right I need some sort of summary here.